go/rpcclientlib/obscuro_client.go

```diff
@@ -19,39 +28,264 @@ const (
 	RPCAddViewingKey        = "obscuro_addViewingKey"
 	RPCStopHost             = "obscuro_stopHost"
 	RPCCall                 = "eth_call"
+	RPCGetBalance           = "eth_getBalance"
+	RPCGetTxCount           = "eth_getTransactionCount"
 	RPCGetTxReceipt         = "eth_getTransactionReceipt"
 	RPCSendRawTransaction   = "eth_sendRawTransaction"
 	RPCGetTransactionByHash = "eth_getTransactionByHash"
+
+	// todo: this is a convenience for testnet testing and will eventually be retrieved from the L1
+	enclavePublicKeyHex = "034d3b7e63a8bcd532ee3d1d6ecad9d67fca7821981a044551f0f0cbec74d0bc5e"
+
+	// ViewingKeySignedMsgPrefix is the prefix added when signing the viewing key in MetaMask using the personal_sign
+	// API. Why is this needed? MetaMask has a security feature whereby if you ask it to sign something that looks like
+	// a transaction using the personal_sign API, it modifies the data being signed. The goal is to prevent hackers
+	// from asking a visitor to their website to personal_sign something that is actually a malicious transaction (e.g.
+	// theft of funds). By adding a prefix, the viewing key bytes no longer looks like a transaction hash, and thus get
+	// signed as-is.
+	ViewingKeySignedMsgPrefix = "vk"
```

Comment:
@BedrockSquirrel Can you review these constants please? Some are literally unused now, some are public but only used in this file.